hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/ITestS3GuardTool.java

```diff
@@ -71,6 +72,7 @@ public void testLandsatBucketRequireGuarded() throws Throwable {
   @Test
   public void testLandsatBucketRequireUnencrypted() throws Throwable {
     skipIfClientSideEncryption();
+    skipIfAnyEncryptionSet(getConfiguration());
```

Comment:
instead of skipping, how about `removeBaseAndBucketOverrides(getConfiguration(), Constants.S3_ENCRYPTION_ALGORITHM);` and you can remove the `skipIfClientSideEncryption()`  too?

Gets better coverage even if encryption settings are used.

Reply:
Nice, it would be better coverage regardless of encryption configured. Tested the changes with and without encryption.
Thanks @ahmarsuhail